rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+#channel = "stable"
+channel = "nightly-2024-04-23"
```

Comment:
Why nightly? :)

Reply:
The original binaries used in probe-rs where built with this version. Updating this will be the next step.

Reply:
Ah makes perfect sense :)